.github/workflows/trigger_rebase.yml

```diff
@@ -0,0 +1,42 @@
+name: Trigger Rebase
+
+on:
+  push:
+    branches:
+      - develop
+env:
+  AUTO_REBASE_PERSONAL_ACCESS_TOKEN: ${{ secrets.AUTO_REBASE_PERSONAL_ACCESS_TOKEN }}
```

Comment:
Where does the "top level" env var get used, if anywhere?

The job steps get the token again from `secrets`, so they don't appear to use the top level var.

Reply:
Oooh, well spotted! 👍